infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServerNode.java

```diff
@@ -174,6 +175,11 @@ public ChannelFuture start() {
             }
         });
 
+        MicroMeterUtils.gauge("netty.usedHeapMemory", PooledByteBufAllocator.DEFAULT, pool ->
+                pool.metric().usedHeapMemory());
+        MicroMeterUtils.gauge("netty.usedDirectMemory", PooledByteBufAllocator.DEFAULT, pool ->
+                pool.metric().usedDirectMemory());
```

Comment:
Do we want to included unpooled metrics as well? A lot of our allocations are done with `Unpooled.buffer()` which uses a different allocator `UnpooledByteBufAllocator.DEFAULT`. It could be helpful to have both sets of statistics.

`ALLOC = UnpooledByteBufAllocator.DEFAULT;`

Reply:
good point.